scripts/flash_klipper.sh

```diff
@@ -52,16 +52,57 @@ function init_flash_process() {
     esac
   done
 
-  ### step 2: select how the mcu is connected to the host
+  ### step 2: select how the mcu is flashed (flash/serialflash)
+  select_flash_command
+
+  ### step 3: select how the mcu is connected to the host
   select_mcu_connection
 
-  ### step 3: select which detected mcu should be flashed
+  ### step 4: select which detected mcu should be flashed
   select_mcu_id "${method}"
 }
 
 #================================================#
 #=================== STEP 2 =====================#
 #================================================#
+function select_flash_command() {
+  unset flash_command
+
+  top_border
+  echo -e "| How to flash MCU?                                     |"
+  echo -e "| 1) make flash (default)                               |"
+  echo -e "| 2) make serialflash (stm32flash)                      |"
+  blank_line
+  back_help_footer
+
+  local choice
+  while true; do
+    read -p "${cyan}###### Flashing command:${white} " choice
```

Comment:
Option "1" needs to be added as the default here, if we want to pre-fill the input with "1".

Reply:
Added default option